share/src/main/java/org/alfresco/web/scripts/UsersWebScript.java

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
@alexbalmus I find the name of this webscript confusing. I can't see this web script being related to users.